request_test.go

```diff
@@ -2,6 +2,7 @@ package sarama
 
 import (
 	"bytes"
+	"reflect"
```

Comment:
We have an HTTP proxy running in production that uses reflection for some things and did not notice any issues with performance whatsoever, so I see no reasons to worry about bringing reflection package into action even in production.